src/Http/Routing/src/DataSourceDependentCache.cs

```diff
@@ -18,6 +18,9 @@ internal class DataSourceDependentCache<T> where T : class
         private object _lock;
```

Comment:
I would also mark this as `readonly`

Reply:
This is used with LazyIntialializer, it can't be readonly.